### PR TITLE
#1221 - add options to extend menu links

### LINKED ISSUE #1221: [connect > portal] - Add option to extend menu options

### PR DESCRIPTION
**Pull request by sebastianscatularo**
image

image

## Sample Config

```json
{
  "showHamburgerMenu": false,
  "menu": [
    {
      "label": "Advance Tools",
      "href": "https://portalbridge.com",
      "target": "_self",
      "order": 1
    }
  ]
}
```

### CODE REVIEW

File: wormhole-connect/README.md [hunk 1]
```diff
@@ -82,3 +82,36 @@ class WormholeConnect extends React.Component {
   }
 }
 ```
+
+## Togle Hamburguer Menu
+
+by setting `showHamburgerMenu` to **false**, you can toggle hamburguer menu off
+and that makes the links to be placed at bottom
```
Comment (kev1n-peters) [inline wormhole-connect/README.md]: By setting the `showHamburgerMenu` option to **false**, you can deactivate the hamburger menu, causing the links to be positioned at the bottom.

File: wormhole-connect/README.md [hunk 2]
```diff
@@ -82,3 +82,36 @@ class WormholeConnect extends React.Component {
...
+
+### Add extra menu entry
+
+by setting `showHumburgerMenu` to **false**, you can use `menu` array to add
+extra links
```
Comment (kev1n-peters) [inline wormhole-connect/README.md]: By setting the `showHumburgerMenu` option to **false**, you can use the `menu` array to add extra links.

File: wormhole-connect/README.md (hunk 1 above)
Comment (sebastianscatularo) [reply wormhole-connect/README.md]: thanks, updated

File: wormhole-connect/README.md (hunk 2 above)
Comment (sebastianscatularo) [reply wormhole-connect/README.md]: thanks, updated

File: wormhole-connect/README.md [hunk 3]
```diff
@@ -82,3 +82,34 @@ class WormholeConnect extends React.Component {
   }
 }
 ```
+
+## Togle Hamburguer Menu
```
Comment (M-Picco) [inline wormhole-connect/README.md]: `Togle` -> `Toggle`

File: wormhole-connect/README.md [hunk 4]
```diff
@@ -82,3 +82,34 @@ class WormholeConnect extends React.Component {
...
+
+By setting the `showHamburgerMenu` option to false, you can deactivate the hamburger menu, causing the links to be positioned at the bottom.
+
+### Add extra menu entry
+
+By setting the `showHumburgerMenu` option to false, you can use the menu array to add extra links.
```
Comment (M-Picco) [inline wormhole-connect/README.md]: `showHumburgerMenu` -> `showHamburgerMenu`

Comment (sebastianscatularo) [reply wormhole-connect/README.md]: updated, thanks

File: wormhole-connect/README.md (hunk 3 above)
Comment (sebastianscatularo) [reply wormhole-connect/README.md]: updated, thanks

**Review by M-Picco — APPROVED** [review]

**Pull request merged**